### ISSUE TITLE
Is it possible to accumulate gradients over multiple batches?

### ISSUE THREAD

**Issue by krumo** [facebookresearch/Detectron#562]
Due to the limitation of GPUs, I cannot train the Network with a large batch size. Is there any way to accumulate gradients over multiple batches to mimic the gradient updates in multiple GPU settings? I remember there's a solution in Caffe but I cannot find any similar solutions in Caffe2 and Detectron. Thanks in advance!

**Comment by ir413** [comment]
Hi @krumo, gradient accumulation is not supported at this time but can be implemented in Detectron. Sorry for the inconvenience.
